packer/windows-vhd-builder.json

```diff
@@ -0,0 +1,67 @@
+{
+    "variables": {
+        "client_id": "{{env `AZURE_CLIENT_ID`}}",
+        "client_secret": "{{env `AZURE_CLIENT_SECRET`}}",
+        "tenant_id": "{{env `AZURE_TENANT_ID`}}",
+        "subscription_id": "{{env `AZURE_SUBSCRIPTION_ID`}}"
+    },
+    "builders": [
+        {
+            "type": "azure-arm",
+            "client_id": "{{user `client_id`}}",
+            "client_secret": "{{user `client_secret`}}",
+            "tenant_id": "{{user `tenant_id`}}",
+            "subscription_id": "{{user `subscription_id`}}",
+            "location": "westus2",
+            "vm_size": "Standard_D4s_v3",
+            "os_type": "Windows",
+            "image_publisher": "MicrosoftWindowsServer",
+            "image_offer": "WindowsServer",
+            "image_sku": "2019-Datacenter-Core-smalldisk",
+            "image_version": "17763.615.1907121548",

```

Comment:
should this be a variable so we can update it in the pipeline instead of having to merge a PR everytime it changes? Or do we want to repo check in so that we can have PRs + reviews for version bumps?

Reply:
I think we should have repo checkins to get reviews on version bumps. 
@PatrickLang for input as well.

Reply:
I will move a few turn a few more fields like location and vs size into variables tho.